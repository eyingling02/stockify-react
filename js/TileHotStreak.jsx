const React = require('react')
const TileHotStreak = (props) => (
    <div className='outer'>
        <div className='buy'>
            <a href='javascript:;'>BUY</a>
        </div>
        <div className='inner'>
            <a href='livingroom.html' target='_blank'>
                <h2>{props.Ticker}</h2>
                <h3>{props.StreakInDays} predictions!</h3>
                <h3>{props.PredictionCount} days</h3>
                <h3>{props.Username}</h3>
            </a>
        </div>
        <div className='sell'>
            <a href='javascript:;'>SELL</a>
        </div>
    </div>
)

const {string} = React.PropTypes

// Tile.propTypes = {
//   title: string.isRequired,
//   description: string.isRequired,
//   year: string.isRequired,
//   poster: string
// }

module.exports = TileHotStreak
