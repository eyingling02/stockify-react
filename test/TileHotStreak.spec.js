import React from 'react'
import TileHotStreak from '../js/TileHotStreak.jsx'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

test('HotStreak snapshot test', () => {
  const component = shallow(<TileHotStreak />)
  const tree = shallowToJson(component)
  expect(tree).toMatchSnapshot()
})
