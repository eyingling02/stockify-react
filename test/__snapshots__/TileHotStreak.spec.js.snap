exports[`test HotStreak snapshot test 1`] = `
<div
  className="outer">
  <div
    className="buy">
    <a
      href="javascript:;">
      BUY
    </a>
  </div>
  <div
    className="inner">
    <a
      href="livingroom.html"
      target="_blank">
      <h2 />
      <h3>
         predictions!
      </h3>
      <h3>
         days
      </h3>
      <h3 />
    </a>
  </div>
  <div
    className="sell">
    <a
      href="javascript:;">
      SELL
    </a>
  </div>
</div>
`;
